cmd/clusterimageset-updater/main.go

```diff
@@ -65,6 +75,25 @@ func main() {
 		logrus.WithError(err).Fatal("Invalid option")
 	}
 
+	if err := addSchemes(); err != nil {
+		logrus.WithError(err).Fatal("Failed to set up scheme")
+	}
+
+	s := json.NewSerializerWithOptions(json.DefaultMetaFactory, scheme.Scheme,
+		scheme.Scheme, json.SerializerOptions{Yaml: true, Pretty: false, Strict: false})
+
+	if err := filepath.WalkDir(o.poolDir, func(path string, info fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() || !strings.HasSuffix(info.Name(), "_clusterpool.yaml") {
```

Comment:
Is this really the best way to detect a clusterpool definition? as opposed to maybe some golang equivalent of `grep 'Kind: ClusterPool'`?

Reply:
It has been used in other places of this tool already and it turned to be OK.
The damage is very small even if it is not accurate.  :)